ray-operator/Makefile

```diff
@@ -81,8 +81,8 @@ build: generate fmt vet ## Build manager binary.
     "                                           \
     -o bin/manager main.go
 
-run: manifests generate fmt vet ## Run a controller from your host.
-	go run ./main.go
+run: manifests generate fmt vet ## Run the controller from your host.
```

Comment:
Why do we need this in case the KubeRay operator has to communicate with the Ray head when using RayJob or RayService?

Reply:
You mean why we need to do `make run` ? 
It starts the kuberay operator locally as if it were deployed to a cluster. I think basically this lets you re-built and re-start the operator quickly.

Reply:
> KubeRay operator has to communicate with the Ray head when using RayJob or RayService?

KubeRay operator needs to send HTTP requests to the Ray head when using RayJob or RayService. How does KubeRay communicates with the head Pod If we run the operator locally?

Reply:
good question. I guess this is the reason you don't want people run operator outside the of a cluster. maybe I should remove this instead.

Reply:
> I guess this is the reason you don't want people run operator outside the of a cluster.

Yes. Maybe we can remove it from the Makefile.

Reply:
I will leave a note and commented it out in case someone wants to enable it.

Reply:
It's fine, but I may delete it in the future.